### PR TITLE
Update changelog 4.0.0 [skip ci]

### PR DESCRIPTION
**Pull request by vigneshshanmugam**
+ The breaking changes commit was sqaushed and the commit body was all over the changelog.  Other breaking changes commits did not have commit . body and it wasn't captured in breaking changes 

+ . Updated the master changelog manually till we find a good solution to automate that.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,28 @@
...
+* **rum-core:** Add task API ([#194](https://github.com/elastic/apm-agent-rum-js/issues/194)) ([0153229](https://github.com/elastic/apm-agent-rum-js/commit/0153229))
+* **rum-core:** measure all resource entries in page load ([#173](https://github.com/elastic/apm-agent-rum-js/issues/173)) ([7cd4e0d](https://github.com/elastic/apm-agent-rum-js/commit/7cd4e0d))
+
+
+### Performance Improvements
+
+* **rum-core:** avoid url parsing on resource timing entries ([#174](https://github.com/elastic/apm-agent-rum-js/issues/174)) ([54ea6b9](https://github.com/elastic/apm-agent-rum-js/commit/54ea6b9))
+
+
+### BREAKING CHANGES
```
Comment (jahtalab) [inline CHANGELOG.md]: I think the breaking changes should be at the top!

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,28 @@
...
+
+* move IE 10 and Android 4 to unsupported list (#196) ([16f4440](https://github.com/elastic/apm-agent-rum-js/commit/16f4440)), closes [#196](https://github.com/elastic/apm-agent-rum-js/issues/196)
+* Rename the final JS bundles (#202) ([68b37d](https://github.com/elastic/apm-agent-rum-js/commit/68b37d))
```
Comment (jahtalab) [inline CHANGELOG.md]: Maybe should add these to the rum changelog as well

File: CHANGELOG.md (hunk 1 above)
Comment (vigneshshanmugam) [reply CHANGELOG.md]: This is auto generated, We cannot change the format. Its according to the conventional changelog generator.

File: CHANGELOG.md (hunk 2 above)
Comment (vigneshshanmugam) [reply CHANGELOG.md]: Sure, lets do that.

File: CHANGELOG.md (hunk 1 above)
Comment (jahtalab) [reply CHANGELOG.md]: In almost all changelogs I've seen the breaking changes are at the top of the release notes. see [here](https://github.com/babel/babel/blob/master/CHANGELOG.md#v700-rc4-2018-08-27) for an example.

Didn't you manually copy the change log items here?

Comment (vigneshshanmugam) [reply CHANGELOG.md]: I did manually, but changelog generator doesn't work that way. Having breaking changes in the bottom on other `packages/changelog.md` and having it in the top on `root changelog` doesnt makes sense.

Comment (vigneshshanmugam) [reply CHANGELOG.md]: Check lerna changelog - https://github.com/lerna/lerna/blob/master/CHANGELOG.md#300-2018-08-10

**Pull request merged**